server/testdata/endpoints/18_couper.hcl

```diff
@@ -0,0 +1,32 @@
+server "couper" {
+  endpoint "/abcdef" {
+    proxy = "test"
```

Comment:
This does work with free endpoints, but not with endpoints within an `api` block.

Reply:
Fixed